library/Icinga/Application/Modules/Module.php

```diff
@@ -307,7 +317,7 @@ public function getSearchUrls()
     /**
      * Return this module's dashboard
      *
-     * @return  Navigation
+     * @return Navigation
```

Comment:
This doesn't **have to** return an instance of this. This method is internal, so you can return whatever you see fit for your enhanced dashboards.